services/harmony/test/helpers/jobs.ts

```diff
@@ -174,7 +173,7 @@ export function containsJob(job: JobRecord, jobList: JobListing): boolean {
  * @param query - Mapping of query param names to values
  * @returns The response
  */
-export function jobListing(app: Application, query: object = {}): Test {
```

Comment:
Why are you removing all these type annotations?

Reply:
The types were incorrect - they should be from supertest.App but I could not find a way to load in the supertest types so I needed to remove them to compile successfully.